src/samples/Sample23.CustomMigration/MigrateStub.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Sample23.CustomMigration
```

Comment:
Is there a specific reason you had to create another project just for the migrations? Preferably we can keep the sample code within `Sample23`.

Reply:
The migrations targeting a custom schema are now in the Sample23 project.